### PR TITLE
Local storage

### PR DESCRIPTION
**Pull request by jennydely**
I added an local storage to save and load events from it.

Don't review the CreateEvent Page, there will be an separated request for this page.

Link to the Task: #18

### CODE REVIEW

File: src/common/GlobalStyles.js [hunk 1]
```diff
@@ -29,18 +29,21 @@ export default createGlobalStyle`
     font-size: 112%;
```
Comment (manuelpapa) [inline src/common/GlobalStyles.js:29]: In a later step maybe you can commit to _one_ size-unit, e.g. `rem`. This lets you easily change values globally and provides a better performance on different devices.

**Review by manuelpapa — COMMENTED** [review]
Great job, very consistent and logic code. Keep going! 🚀

File: src/componentsEventPage/EventForm.js [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+                    </button>
+                    <SubmitButton type="submit">Save</SubmitButton>
+                </ButtonGroup>
+            </Form>
+        </>
+    )
+}
+
+const ErrorMessage = styled.p`
+  color: var(--orange-main);
```
Comment (EmilyCook321) [inline src/componentsEventPage/EventForm.js]: Var is somewhat outdated- would you consider using 'let' instead? 
I found the article balanced and helpful. 
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

**Review by EmilyCook321 — COMMENTED** [review]
Aside from the use of var, looking good!

**Pull request closed without merging**